microk8s-resources/actions/disable.reg.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+echo "Disabling Reg"
```

Comment:
I think a single message should be enough. :)

Reply:
Alright!